accounts-db/src/accounts_db.rs

```diff
@@ -1343,6 +1343,19 @@ impl StoreAccountsTiming {
     }
 }
 
+#[derive(Default, Debug)]
+struct CleaningInfo {
+    pub slot_list: SlotList<AccountInfo>,
+    pub ref_count: u64,
+}
+
+impl CleaningInfo {
+    pub fn update(&mut self, slot_list: SlotList<AccountInfo>, ref_count: u64) {
```

Comment:
i'd just drop the fn. less code to test? we don't usually do things like this very localized structs.